docker/docker-compose-with-obs.yml

```diff
@@ -0,0 +1,105 @@
+---
+version: "3"
+x-image: &image
+  image: ${RW_IMAGE:-risingwavelabs/risingwave:nightly-20231207}
```

Comment:
Will change to the release version after https://github.com/risingwavelabs/risingwave/pull/13844 is released.

Reply:
you need add https://github.com/risingwavelabs/risingwave/pull/13844 to v1.5.0-rc.